### PR TITLE
Fixed issues

### PR DESCRIPTION
**Pull request by sairamch04**
- fetch orderId immediatly - used session.save 
  added function in DAO
- Locality,Buyer,Apartment Service insert method changed, So incorporated those changes

### CODE REVIEW

File: src/main/java/com/foodybuddy/dao/OrderDAO.java [hunk 1]
```diff
@@ -12,10 +13,18 @@
...
+	public Order insert(Order order);
+	
+	/**
+	 * Save. Returns the Identifier immediately
+	 *
+	 * @param order the order
+	 * @return the order
 	 */
-	public Order insert(Order c);
+	public Order save(Order order);
```
Comment (SainathB) [inline src/main/java/com/foodybuddy/dao/OrderDAO.java:26]: Why did you add save again? insert is there right?

Comment (sairamch04) [inline src/main/java/com/foodybuddy/dao/OrderDAO.java:26]: This saves the order immediately and give the orderId . We don't need to wait till the transaction completed. This particular scenario is used in placing order

Comment (SainathB) [inline src/main/java/com/foodybuddy/dao/OrderDAO.java:26]: Ok.
